server/controllers/list.js

```diff
@@ -1513,6 +1512,165 @@ const leaveList = (req, resp) => {
     });
 };
 
+const getListsForItem = (req, resp) => {
+  const {
+    user: { _id: userId }
+  } = req;
+
+  const query = {
+    memberIds: userId,
+    isArchived: false
+  };
+
+  List.find(query, 'name')
+    .lean()
+    .exec()
+    .then(docs => {
+      if (!docs) {
+        return resp.sendStatus(400);
+      }
+
+      return resp.send(docs);
+    })
+    .catch(() => resp.sendStatus(400));
+};
+
+const moveItem = (req, resp) => {
+  const { itemId, newListId } = req.body;
+  const { id: listId } = req.params;
+  const { _id: userId } = req.user;
+  const sanitizedItemId = sanitize(itemId);
+  const sanitizedListId = sanitize(listId);
+  const sanitizedNewListId = sanitize(newListId);
+
+  List.findOne({
+    _id: sanitizedListId,
+    memberIds: userId,
+    'items._id': sanitizedItemId
+  })
+    .exec()
+    .then(oldList => {
+      if (!oldList) {
+        throw new BadRequestException();
+      }
+
+      const { items } = oldList;
+      const {
+        authorId,
+        createdAt,
+        description,
+        editedBy,
+        isArchived,
+        isDeleted,
+        isOrdered,
+        name,
+        purchaserId,
+        status,
+        updatedAt,
+        voterIds
+      } = items.id(sanitizedItemId);
+      const newItem = new Item({
+        authorId,
+        createdAt,
+        description,
+        editedBy,
+        isArchived,
+        isDeleted,
+        isOrdered,
+        locks: { description: false, name: false },
+        name,
+        purchaserId,
+        status,
+        updatedAt,
+        voterIds
+      });
```

Comment:
`const newItem = new Item({ ...items.id(sanitizedItemId).lean(), locks: { description: false, name: false }})`?

Reply:
It can't be done with `lean()` but I fix this in other way:
```
const { items } = oldList;
      const newItem = new Item({
        ...items.id(sanitizedItemId)._doc,
        locks: { description: false, name: false }
      });
```